script/genkeys

```diff
@@ -13,13 +13,23 @@ KEYMASTER="docker run --volume ${ROOT}/certificates:/certificates --rm cloudpipe
 if [ ! -f "${ROOT}/certificates/password" ]; then
   echo ">> creating a random password in ${ROOT}/certificates/password."
   touch ${ROOT}/certificates/password
-  chmod 600 ${ROOT}/certificates/password
+  chmod 640 ${ROOT}/certificates/password
   # "If the same pathname argument is supplied to -passin and -passout arguments then the first
   # line will be used for the input password and the next line for the output password."
-  cat /dev/random | head -c 128 | base64 > ${ROOT}/certificates/password
+  cat /dev/urandom | head -c 128 | base64 > ${ROOT}/certificates/password
```

Comment:
> You do realize that's the same document I referenced in my commit message? :)

Wow, that's rich. I did not look at your full commit message. :D We're in tune there then.

The reason I err on the side of `/dev/random` is the fear that people are going to launch it directly with the dev setup and leave it running anyway.

With all that being said, I agree with your sentiments and will definitely merge this as is.

Thank you again.